src/Access/AccessRights.cpp

```diff
@@ -61,12 +61,21 @@ namespace
                     res.any_database = true;
                     res.any_table = true;
                     res.any_column = true;
+                    res.any_global_with_parameter = true;
                     break;
                 }
                 case 1:
                 {
-                    res.any_database = false;
-                    res.database = full_name[0];
+                    if (access_flags.isGlobalWithParameter())
+                    {
+                        res.any_global_with_parameter = false;
+                        res.parameter = full_name[0];
```

Comment:
It looks inconsistent that in this case you make `any_column == true` and `any_table==true`.

Reply:
fixed - made this only if `!access_flags.isGlobalWithParameter()`